### DISCUSSION TITLE
Help about a remembered key stroke

### DISCUSSION THREAD

**Discussion by halloleo** [saulpw/visidata#2228] [Ideas]
VisiData comes with a good built-in documentation listing long commands and key shortcuts. (`z ^h`)

Sometimes however it could be nice to type a key stroke and get a help on what only this key stroke would do. 

Emacs offers this with `^h k` followed by the key stroke. So, for example `^h k ^x ^s` brings up the help about the `save-buffer` command (initiated via keyboard by ` ^x ^s`). 

This is pretty invaluable _when you quickly want to check whether your memory about a command is correct_.

Just an idea...

**Comment by saulpw** [comment]
Interesting idea, @halloleo !  I don't think we can use `Ctrl+H` for this; what keystroke would you suggest?

**Reply by Mickychen00** [reply]
Like the WhichKey Plugin in Vim/Neovim?

**Reply by halloleo** [reply]
@saulpw Of course, Ctrl-H is taken. Not sure what would fit with the carefully crafted visidata key scheme... Maybe just a long-command users can bind?

@Mickychen00 I don't know vim well, but often I have a keye combo in mind, but want to doublecheck whether it really does what I want...

**Reply by daviewales** [reply]
It could be a long command, like `:help` in vim.

For example, in Vim I can type either `:help ^f` or `:help ctrl-f` to learn more about the ctrl-f command.
However, this requires you to type the symbolic form of the command, rather than the command itself.
Perhaps another long command `help-interactive` could pop up a dialog and wait for a key press, to provide help directly for shortcuts without needing to type them in symbolically?
